relay-metrics/src/aggregatorservice.rs

```diff
@@ -274,11 +346,47 @@ impl AggregatorService {
             Aggregator::MergeBuckets(msg) => self.handle_merge_buckets(msg),
             #[cfg(test)]
             Aggregator::BucketCountInquiry(_, sender) => {
```

Comment:
`BucketCountInquiry` was used to test a single aggregator service, now that we got rid of it, i changed it to test the default aggregator, it's only used for testing.